integration/specs/CodeInput/codeInput-1.spec.js

```diff
@@ -0,0 +1,107 @@
+const PageCodeInput = require('../../../src/components/CodeInput/pageObject');
+const { DELETE_KEY, TAB_KEY } = require('../../constants');
+
+const CODEINPUT = '#codeinput-1';
+
+describe('CodeInput base example', () => {
+    beforeAll(() => {
+        browser.url('/#!/CodeInput/1');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(CODEINPUT);
+        component.waitForExist();
+    });
+    it('should have focus on the first input on load', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        expect(codeInput.getFocusedIndex()).toBe(0);
+    });
+    it('should change focus to the second input when the first input its filled with a number', () => {
+        const codeInput = new PageCodeInput(CODEINPUT);
+        browser.keys(['', '2']);
```

Comment:
why we need to pass ```['', '2']```?

Reply:
My bad: I though it was necessary the first value, it refers to Metadata, I changed to key only and seems to be working...

- - - - -

Reverted to browsers.keys('2'); all of them in next PR update

-